src/mz/src/utils.rs

```diff
@@ -13,9 +13,66 @@ use indicatif::{ProgressBar, ProgressStyle};
 
 use crate::ExitMessage;
 
-/**
- * Trim lines. Useful when reading input data.
- */
+use clap::ArgEnum;
+
+/// Cloud providers and regions available.
+#[derive(Debug, Clone, ArgEnum)]
+pub(crate) enum CloudProviderRegion {
+    AwsUsEast1,
+    AwsEuWest1,
+}
+
+/// Implementation to name the possible values and parse every option.
+impl CloudProviderRegion {
+    pub fn variants() -> [&'static str; 2] {
+        ["aws/us-east-1", "aws/eu-west-1"]
+    }
+    pub fn parse(region: String) -> CloudProviderRegion {
+        match region.as_str() {
+            "aws/us-east-1" => CloudProviderRegion::AwsUsEast1,
+            "aws/eu-west-1" => CloudProviderRegion::AwsEuWest1,
+            _ => panic!("Unknown region."),
```

Comment:
This method would be much better as [a FromStr impl](http://doc.rust-lang.org/1.63.0/std/str/trait.FromStr.html) (and then you can remove the `panic` call, which makes me a tiny bit nervous).